### ISSUE TITLE
t_spawn_item

### ISSUE THREAD

**Issue by xMirroRx** [Sphereserver/Source-X#1218]
Items created with t_spawn_item take the decay timer in sphere.ini and are deleted from after save when it reaches -1. This shouldn't happen.

**Comment by xwerswoodx** [comment]
I see what you mean, I don't think it's hard thing to fix but the problem is I am not sure if this is a bug or not, because I think they should decay in time and respawn after it's time ticked, but we may add some sort of FLAG in sphere.ini for it.

**Comment by Jhobean** [comment]
> I see what you mean, I don't think it's hard thing to fix but the problem is I am not sure if this is a bug or not, because I think they should decay in time and respawn after it's time ticked, but we may add some sort of FLAG in sphere.ini for it.

If item is link with spawn point they should not decay.

**Comment by Jhobean** [comment]
Optimisation can be done and item generate with spawn point could have timer -1 direct without attr_decay

**Comment by xwerswoodx** [comment]
I said this because it's intentionally set decayed in source;

```    pItem->SetDecayTime(g_Cfg.m_iDecay_Item);	// it will decay eventually to be replaced later```

So I can add some flag for blocking to set decay time if someone doesn't want them to decay.

**Comment by xMirroRx** [comment]
Yes, I found exactly such a solution. Actually, the reason why I posted this here was to understand whether it was a mistake or not, and it was also annoying because when a different item was linked to the items created with t_spawn_item, all of these linked items gave warnings during saving. I wanted to let you know.

**Comment by cbnolok** [comment]
Is this still persisting? Should have been addressed by a fix commit.

**Comment by AtillaG1** [comment]
Yes for me its fine now.
